ansible_runner/runner.py

```diff
@@ -121,6 +155,15 @@ def run(self):
             if k != 'PATH' and isinstance(v, six.text_type):
                 self.config.env[k] = v.encode('utf-8')
 
+        # Prepare to collect performance data
+        if self.config.resource_profiling:
+            cgroup_path = '{0}/{1}'.format(self.config.resource_profiling_base_cgroup, self.config.ident)
+            cmd = 'cgcreate -g cpuacct,memory,pids:{}'.format(cgroup_path)
+            rc = subprocess.call(cmd, shell=True)
+            if rc:
+                # Unable to create cgroup, disable profiling
```

Comment:
We should emit a `warn` logging statement here.

Reply:
done